chapters/solutions.Rmd

```diff
@@ -1582,7 +1654,53 @@ most items would be checked off of the list.
 
 ### Exercise \@ref(packaging-ex-data) {-}
 
-FIXME
+The central idea is that a Python file can find out what directory it was read from,
+so it can look for other files in that directory.
+This means that we don't have assume the package has been installed in a particular place
+or that some environment variable will tell us where to look for our data.
+To test this idea out:
+
+1.  Create a directory called `data-package`.
```

Comment:
In the solution we probably need to describe the full directory tree for the new package:
```
data-package
├── data-package
|   ├── collated.csv
│   └── collated.py
├── README.md
└── setup.py
```
as well as the contents of `setup.py`:
```
from setuptools import setup


setup(
    name='data-package',
    version='0.1',
    author='Amira Khan',
    packages=['data-package'],
)
```